client/packages/programs/src/JsonForms/components/ViralLoadStatus.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect, useState } from 'react';
+import {
+  rankWith,
+  ControlProps,
+  uiTypeIs,
+  composePaths,
+} from '@jsonforms/core';
```

Comment:
The component looks good as far as I can see! However, I am trying to keep the components as generic as possible and only if really necessary make them program/HIV specific.

In this case I think you can use the decision tree component as it is. There is already a `greaterThan` condition (`packages/programs/src/JsonForms/components/DecisionTree/DecisionTree.tsx`). 

Could you please try to use this component instead? See the RiskBehaviour as an example. Here you only need a single decision node. Sorry for the extra work only just had the idea.

Reply:
[a3c1042](https://github.com/openmsupply/open-msupply/pull/1237/commits/a3c1042ba4176458d00c42db809279c2e0865bbb) I added a lessThanOrEqualTo condition for the `Suppressed` status (: